packages/jbrowse-cli/tsconfig.json

```
@@ -1,14 +1,14 @@
 {
-  "compilerOptions": {
-    "declaration": true,
-    "importHelpers": true,
-    "module": "commonjs",
-    "outDir": "lib",
-    "rootDir": "src",
-    "strict": true,
-    "target": "es2017"
-  },
-  "include": [
-    "src/**/*"
-  ]
-}
+    "compilerOptions": {
+      "declaration": true,
+      "importHelpers": true,
+      "module": "commonjs",
+      "outDir": "lib",
+      "rootDir": "src",
+      "strict": true,
+      "target": "es2017"
+    },
+    "include": [
+      "src/**/*"
+    ]
+  }
```

Just a tiny nitpick, but it would probably be good to restore the original spacing here.

```suggestion
  "compilerOptions": {
    "declaration": true,
    "importHelpers": true,
    "module": "commonjs",
    "outDir": "lib",
    "rootDir": "src",
    "strict": true,
    "target": "es2017"
  },
  "include": [
    "src/**/*"
  ]
}
```